sdk/core/azure-core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.11.0-beta.1 (Unreleased)
 
+### New Features
+
+- Moved `BinaryData` from `azure-core-experimental` to `azure-core`.
```

Comment:
I would change this to the following as it is net new functionality for `azure-core`.

```suggestion
- Added `BinaryData` which allows for a format agnostic representation of binary information.
```

(Or something along those lines)